mkdocs.yml

```
@@ -35,6 +35,7 @@ nav:
     - Autoscaling: guidance/autoscaler.md
     - Ingress: guidance/ingress.md
     - Observability: guidance/observability.md
+    - Kuberay with MCAD: guidance/kuberay-with-MCAD.md
```

Last nit:
The "official" capitalization is KubeRay
```suggestion
    - KubeRay with MCAD: guidance/kuberay-with-MCAD.md
```